internal/getproviders/package_authentication_test.go

```diff
@@ -365,13 +365,18 @@ func TestSignatureAuthentication_success(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			// Location is unused
 			location := PackageLocalArchive("testdata/my-package.zip")
+			//
+			//providerSource, err := tfaddr.ParseProviderSource("testdata/my-package.zip")
```

Comment:
Did you want to use it in the tests? None of the `NewSignatureAuthentication` invocations test passing a provider source.

Reply:
For now I was going to keep the tests simple and not use it in the tests.